service/controller/v2/resource/deployment/resource.go

```diff
@@ -21,8 +20,6 @@ import (
 const (
 	// Name is the identifier of the resource.
 	Name = "deploymentv2"
-
-	createTimeout = 30 * time.Minute
```

Comment:
There are certain variables/constants that are not used at all as it seems. Boyscouting them away. Swoooosh.